web-apps/genesys-spark-examples/src/utils/generate-icon-list-html.js

```diff
@@ -0,0 +1,23 @@
+const { globSync } = require('glob');
+
+exports.generateIconListHTML = function generateIconListHTML(paths) {
+  const iconNames = globSync(paths)
+    .map(file => {
+      console.log(file);
+      return file.split('gux-icon/icons/').pop().replace('.svg', '');
+    })
+    .sort();
+
+  let output = '';
+
+  output += '<div class="icons-container">';
+  iconNames.forEach(iconName => {
+    output += `<div class="icon-example">
+  <gux-icon size="large" icon-name="${iconName}" decorative="true"></gux-icon>
+  <div class="icon-name">${iconName}</div>
+</div>`;
+  });
```

Comment:
Indentation looks a little off.

Reply:
I know it looks a bit off but because it is a template string it is technically right.